test/index.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
+const dirtyChai = require('dirty-chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+
+chai.use(dirtyChai);
+
+describe('canvas-cover-image', function () {
+  it('If image and canvas are the same size, a straight overlay occurs', function () {
+    const cover = require('..');
+    const ctx = {
```

Comment:
could probably create a function to make the context object as well... although not entirely necessary